helpers.go

```diff
@@ -228,7 +229,7 @@ func getGroupPath(prefix, path string) string {
 // acceptsOffer This function determines if an offer matches a given specification.
 // It checks if the specification ends with a '*' or if the offer has the prefix of the specification.
 // Returns true if the offer matches the specification, false otherwise.
-func acceptsOffer(spec, offer string) bool {
+func acceptsOffer(spec, offer, _ string) bool {
```

Comment:
@nickajacks1 why is this adjustment needed ?

Reply:
This function is passed as an argument to getOffer. This third arg is for the media type parameters, which aren't used here.